oxide-vpc/src/api.rs

```diff
@@ -47,7 +47,7 @@ pub struct BoundaryServices {
     pub mac: MacAddr,
 }
 
-/// The IPv4 configuration for an OPTE port.
+/// The IPv4 configuration of a VPC guest..
```

Comment:
Nit, double periods at the end of the sentence.

Reply:
Done.